src/Data/Jadwal.php

```diff
@@ -170,8 +182,11 @@ public function getUASes($refetch = false)
     }
 
 
-
-    protected function fetch($endpoint)
+    /**
+     * @param $endpoint
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    protected function fetch($endpoint) : void
```

Comment:
This function actually returns mixed, not `void`.

Reply:
Oh lol, my bad, this was my 2019 code, where my standard still a garbage lol. You guessed it right. Sorry.